content/dstu2/immunization.md

```diff
@@ -20,12 +20,25 @@ Search for Immunizations that meet supplied query parameters:
 ### Parameters
 
  Name      | Required? | Type          | Description
------------|-----------|---------------|----------------------------------------------------------
+-----------|-----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  `patient` | Y         | [`reference`] | The patient for the vaccination record. Example: `12345`
+ `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'. Single dates are treated as implied ranges. EG: `date=ge2014-09-24` `&date=le2015-09-24`
```

Comment:
Remove the last sentence, we don't treat single dates as ranges. That would be like date=2014-09-24, that would mean we return all the immunizations for that day.

Also, this description sounds like both dates are required. Can you indicate that one date works?